docs/reference/settings/ml-settings.asciidoc

```diff
@@ -100,6 +100,13 @@ dynamic setting in version 7.1. As a result, changes to its value after node
 startup are used only after every node in the cluster is running version 7.1 or
 higher. The maximum permitted value is `512`.
 
+`xpack.ml.nightly_maintenance_requests_per_second`::
```

Comment:
@szabosteve can you review this do update please. The setting has been around for a while we must have missed documenting it.